Sources/Conference/Components/SpeakerDetail.swift

```diff
@@ -0,0 +1,62 @@
+import Plot
+
+struct SpeakerDetail: Component {
+    let speaker: Speaker
+    
+    var body: Component {
+        Div {
+            Div().id("particles-js")
+            Div {
+                Div {
+                    Div {
+                        Image(url: speaker.image, description: speaker.name)
+                    }
+                    .class("image")
+                    Div {
+                        Div {
+                            H3(speaker.name)
+                            Div {
+                                if speaker.github != nil {
+                                    Link(url: speaker.github!) {
+                                        Span {
+                                            SVG(url: "/App/YearX/images/icons/feather-sprite.svg#github").class("feather")
+                                        }.class("icon")
+                                    }
+                                }
+                                if speaker.github != nil {
+                                    Link(url: speaker.twitter!) {
```

Comment:
This should be Twitter
```suggestion
                                if let twitterURL = speaker.twitter {
                                    Link(url: twitterURL) {
```